tests/components/history_stats/test_sensor.py

```diff
@@ -496,6 +619,7 @@ def _fake_states(*args, **kwargs):
                         "start": "{{ as_timestamp(utcnow()) - 3600 }}",
                         "end": "{{ utcnow() }}",
                         "type": "time",
+                        "unique_id": "6b1f54e3-4065-43ca-8492-d0d4506a573a",
```

Comment:
I'll guess this was probably just forgotten in the past, otherwise this sensor is identical to the one above it, and there's no reason for that. All the other places w/ duplicated sensors have one w/ unique_id and one without.